backend/tests/Tests/Authorization.Tests.fs

```diff
@@ -39,36 +22,4 @@ let testSetUserAccess =
           name = "Dark Backend Test Admin" }
       |> Task.map (Exception.unwrapResultInternal [])
 
-    let username = UserName.create "test2"
-    let owner = OwnerName.create "test_admin2"
-
-    do! A.setUserAccess username owner None
-    let! permission = A.grantedPermission username owner
-    Expect.equal permission None "no initial access"
-
-    do! A.setUserAccess username owner (Some A.Read)
-    let! permission = A.grantedPermission username owner
-    Expect.equal permission (Some A.Read) "read access"
-
-    do! A.setUserAccess username owner (Some A.ReadWrite)
-    let! permission = A.grantedPermission username owner
-    Expect.equal permission (Some A.ReadWrite) "write access"
-
-    do! A.setUserAccess username owner None
-    let! permission = A.grantedPermission username owner
-    Expect.equal permission None "revoked access"
   }
-
-
-let testPermissionMax =
-  test "permissionMax" {
-    Expect.equal (A.Read.max (A.Read)) A.Read ""
-
-    Expect.equal (A.Read.max (A.ReadWrite)) A.ReadWrite ""
-
-    Expect.equal (A.ReadWrite.max (A.Read)) A.ReadWrite ""
-
-    Expect.equal (A.ReadWrite.max (A.ReadWrite)) A.ReadWrite ""
-  }
-
-let tests = testList "A" [ testPermissionMax; testSetUserAccess ]
```

Comment:
I've realized that I accidentally removed both tests here. Should I add `testSetUserAccess` back and remove it in a separate PR where I start removing things from `Accounts.fs`, or should I just delete the `Authorization.Tests.fs `file?